apis/services/profileService.js

```diff
@@ -0,0 +1,21 @@
+const { profileDao } = require('../models');
+
+const getCreatedList = async (userID) => {
+  const userInfo = await profileDao.getUserInfo(userId);
+  const createdList = await profileDao.getCreatedList(userId);
+
+  const result = { userInfo, createdList };
+  return result;
```

Comment:
```suggestion
  return { userInfo, createdList };
```
return에 대한 코드를 아래 getStoredList function과 통일 시켜주세요. 현재와 같은 경우에는 변수 선언 뒤 할당한 후 해당 변수를 Return 하는 것 보다 깔끔하고 읽기 쉬울 것 같네요!

Reply:
수정